src/SensioLabs/Melody/Melody.php

```diff
@@ -71,6 +71,14 @@ public function run($filename, array $arguments, RunConfiguration $configuration
 
         $cliExecutor($process, false);
     }
+    
+    /**
+     * @return string Returns the base-path used for temp and working files
+     */
+    public function getStoragePath()
+    {
+        return sprintf('%s/melody', sys_get_temp_dir());
```

Comment:
I would set the value inside a class property to avoid a new computation for each call.